From 5d9d390183845f0f919c1edef18a9b019fda00cf Mon Sep 17 00:00:00 2001
From: Eli Collins <eli@cloudera.com>
Date: Fri, 1 Jun 2012 15:27:44 -0700
Subject: [PATCH 101/795] MR1: HADOOP-8466. hadoop-client POM incorrectly excludes avro.

Reason: Bug
Author: Bruno Mahe
Ref: CDH-6141
(cherry picked from commit 3ff541af551480eceb9c75f34ebd55818bc3a259)
(cherry picked from commit 737cba29b00551cf75f2c148b971ab2561e25317)
---
 cloudera/maven-packaging/hadoop-client/pom.xml |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/cloudera/maven-packaging/hadoop-client/pom.xml b/cloudera/maven-packaging/hadoop-client/pom.xml
index 6b870e6..4cdb26a 100644
--- a/cloudera/maven-packaging/hadoop-client/pom.xml
+++ b/cloudera/maven-packaging/hadoop-client/pom.xml
@@ -108,10 +108,6 @@
         </exclusion>
         <exclusion>
           <groupId>org.apache.avro</groupId>
-          <artifactId>avro</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>org.apache.avro</groupId>
           <artifactId>avro-ipc</artifactId>
         </exclusion>
         <exclusion>
-- 
1.7.0.4

